forte/data/ontology/top.py

```diff
@@ -878,15 +878,36 @@ def image(self):
                 "Cannot get image because image annotation is not "
                 "attached to any data pack."
             )
-        return self.pack.get_image_array(self._image_payload_idx)
+        return self.pack.get_payload_data_at(
+            Modality.Image, self._image_payload_idx
+        )
 
     @property
     def max_x(self):
-        return self.image.shape[1] - 1
+        return self._image_width - 1
 
     @property
     def max_y(self):
-        return self.image.shape[0] - 1
+        return self._image_height - 1
+
+    @property
+    def image_shape(self):
+        return (self._image_height, self._image_width)
+
+    def set_image_shape(self, width, height):
```

Comment:
Not sure if it's a good idea to allow flexible image size setting. What if it's in conflict with the size of image at `self._image_payload_idx`?

Reply:
It's only a temporary solution for this PR. There is more implementation needs to be done as this issue suggests. https://github.com/asyml/forte/issues/885